cms/standard_pages/tests/test_models.py

```diff
@@ -0,0 +1,115 @@
+from django.test import TestCase, override_settings
+from wagtail.test.utils import WagtailTestUtils
+
+from cms.standard_pages.tests.factories import IndexPageFactory, InformationPageFactory
+
+
+class IndexPageTestCase(WagtailTestUtils, TestCase):
+    """Test IndexPage model properties and methods."""
+
+    def setUp(self):
+        self.page = IndexPageFactory(
+            title="Test Index Page",
+            description="This is an example",
+            content="This is the main content",
+            featured_items=None,
+            related_links=None,
+        )
+
+        self.page_url = self.page.url
+
+    def test_no_featured_items_displayed_when_no_children_and_no_custom_featured_items_selected(self):
+        """Test that the Featured Items block isn't displayed when the Index Page has no child pages
+        and no custom Featured Items are specified.
+        """
+        self.page.featured_items = None
+
```

Comment:
```suggestion
```

this is extraneous because:
1. the page is set up with `featured_items = None`
2. this changes is only in memory, so it doesn't affect the GET request below

Reply:
[Test improvements 2.0](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/612a1db5994dced67b02a37db7f3c3518885204b)